src/webui/src/components/overview/Progress.tsx

```diff
@@ -19,24 +19,105 @@ interface ProgressProps {
 
 interface ProgressState {
     isShowLogDrawer: boolean;
+    isCalloutVisible?: boolean;
```

Comment:
why `?`?

Reply:
when experiment status is `error`,  hover to the status, will tooltip a modal. 
`isCalloutVisible` decide the modal's hidden | show